server/modifiers.py

```diff
@@ -203,3 +204,98 @@ def insert_empty_statements(self) -> Self:
         self.modified_count += 1
 
         return self
+
+    def reverse_booleans(self) -> Self:
+        """A code modifier that messes up some conditions.
+
+        This will reverse True and False keywords.
+
+        Returns:
+            The modifier instance.
+        """
+        number_boolean_pairs = []
+        for num, line in enumerate(self.file_contents):
+            if any(key in line for key in ["True", "False"]):
+                number_boolean_pairs.extend([(num, key) for key in ["True", "False"] if key in line])
+
+        line_subset = random.sample(number_boolean_pairs, min(self.difficulty, len(number_boolean_pairs)))
+        for num, key in line_subset:
+            self.modified_contents[num] = self.modified_contents[num].replace(key, str(bool(bool(key)-1)))
```

Comment:
This works for when the `key` is "True" but not "False"

Reply:
Fixed

Reply:
Have tested and it works! 🎉